src/components/manage/History/History.jsx

```diff
@@ -10,16 +10,19 @@ import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 import { Container, Dropdown, Icon, Segment, Table } from 'semantic-ui-react';
-import { concat, map, reverse } from 'lodash';
+import { concat, map, reverse, find } from 'lodash';
 import { Portal } from 'react-portal';
 import { FormattedMessage, defineMessages, injectIntl } from 'react-intl';
+import { asyncConnect } from '@plone/volto/helpers';
 
 import {
-  FormattedDate,
+  FormattedRelativeDate,
```

Comment:
Why are we doing this?
We decided that we don't want relative dates in Volto.

Reply:
It's a mistake. No idea how that got here. Reverted...